app/src/main/java/com/example/photogalery/QueryPreferences.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.photogalery
+
+import android.content.Context
+import android.preference.PreferenceManager
+
+private const val PREF_SEARCH_QUERY = "searchQuery"
+
+object QueryPreferences {
+
+    fun getStoredQuery(context: Context): String {
+        val prefs = PreferenceManager.getDefaultSharedPreferences(context)
+        return prefs.getString(PREF_SEARCH_QUERY, "")!!
```

Comment:
We should avoid using force unwrap, because it can produce crashes.

Reply:
Done.